dev/com.ibm.ws.webcontainer.security/test/com/ibm/ws/webcontainer/security/internal/WebAppSecurityConfigImplTest.java

```diff
@@ -465,7 +465,7 @@ public void testGetBasicAuthRealmName_Valid() {
         assertEquals("Vallid value should be returned.", REALM_NAME, webCfg.getBasicAuthRealmName());
     }
 
-    @Test
+    // @Test
```

Comment:
You could set the `com.ibm.ws.beta.edition` JVM property in a `@BeforeClass` method and unset it in an `@AfterClass` method to let these unit tests continue to run instead of commenting them out.